test/tx-builder/dydx-governance.test.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+import hre from 'hardhat';
+
+import { Network, TxBuilder } from '../../src';
+import config from '../../src/config';
+
+describe('DydxGovernance', () => {
```

Comment:
If these tests are only meant to be run on a mainnet fork, can you use [this testing utility function](https://github.com/dydxfoundation/governance-contracts/blob/b06718fe1639d2be4acc02214139dbe6e104a32b/test/helpers/describe-contract.ts#L25-L39) instead?

Example usage [here](https://github.com/dydxfoundation/governance-contracts/blob/b06718fe1639d2be4acc02214139dbe6e104a32b/test/stark-proxy/sp2-guardian.spec.ts#L166-L171).

Example us

Reply:
I dont want to do this - this repo has a lot o really massive and confusing objects and I feel like the bonus is pretty minimal and the obfuscation is really high

Reply:
Not to be pedantic, but it's important that we make sure we're using the correct testing framework as others are (since the current tests you wrote will run in Hardhat test + mainnet fork tests, and potentially break depending on what setup is done beforehand). We also have outside contributors to this repository and I want to hold ourselves to the standards we hold them to.

Additionally, this shouldn't be much additional work (it's a wrapper around the `describe` call that doesn't run in a mainnet fork, in the linked example). You should only have to replace the `describe` call with the same code in the linked example.

Reply:
ok - this is all reasonable!